src/hooks/wishlist-context/wishlist-context.js

```diff
@@ -0,0 +1,23 @@
+import { createContext } from "react";
+import { useState } from "react";
+
+
+const wishlistContext = createContext({itemsInWishlist: 4});
+
+const WishlistProvider = ({children}) => {
+    const [itemsInWishlist, setItems] = useState(0)
+    const addToWishlist = () => {
+        setItems(itemsInWishlist+1)
+    }
+    const removeFromWishlist = () => {
+        setItems(itemsInWishlist-1)
+    }
```

Comment:
you should store product id in wishlist

Reply:
There is no use of product ids **right now**. That's why I haven't stored it.
I will add this functionality in a separate pull request. Thanks for the review.